manifests/blacksmith/shield-backups.yml

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  shield:
+    exodus: (( concat $GENESIS_EXODUS_MOUNT params.shield.deployment_env "/" params.shield.deployment_type ":" ))
+
+    admin_username:  (( vault meta.shield.exodus "admin_username" ))
+    admin_password:  (( vault meta.shield.exodus "admin_password" ))
+
+    address:  (( vault meta.shield.exodus "url" ))
+    agent:    (( vault meta.vault "/shield:agent" )) # TODO: this value is not set by the SHIELD kit.
+
+params:
```

Comment:
Params should always be a single layer.  See https://github.com/genesis-community/genesis/blob/v2.8.x-dev/docs/AUTHORING-KITS.md#user-facing-parameters

So instead of params.shield.username, it should be params.shield_username.  Likewise for the others...